tests/lib/prepare-uc.sh

```diff
@@ -7,10 +7,10 @@ set -x
 . "$TESTSLIB/prepare-utils.sh"
 
 # install dependencies
-install_core22_deps
+install_core24_deps
```

Comment:
Renaming to install_base_deps would be nice

Reply:
Done.